pkg/regressionallowances/regressions_4.17.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+      "JiraComponent": "kube-apiserver",
+      "TestID": "openshift-tests:d6b41cee7afca1c2a0b52f9e6975425f",
+      "TestName": "[bz-kube-apiserver][invariant] alert/KubeAPIErrorBudgetBurn should not be at or above info",
+      "JiraBug": "https://issues.redhat.com/browse/OCPBUGS-42083",
+      "ReasonToAllowInsteadOfFix": "Degraded enough to trip test near the end of stabilization. Still no root cause determined, and the team is still actively investigating. We are marking this as not a blocker because we did not see signals indicating that this does not stabilize after install is completed, although that has not yet been confirmed.",
```

Comment:
```suggestion
      "ReasonToAllowInsteadOfFix": "Degraded enough to trip test near the end of stabilization. This alert fires on installations, when kube-apiserver doesn't get full list of etcds and may lose etcd connectivity for several minutes. We are marking this as not a blocker because it affects fresh installats only and the alert goes away shortly after installation is complete.",
```